checker/src/org/checkerframework/checker/lock/LockAnalysis.java

```diff
@@ -40,4 +43,10 @@ public LockStore createCopiedStore(LockStore s) {
     public CFValue createAbstractValue(AnnotatedTypeMirror type) {
```

Comment:
Why is the superclass of LockAnalysis `CFAbstractAnalysis` instead of `CFAnalysis`.
If this could be changed, these two `createAbstractValue` overrides would no longer be necessary.
If this can't be changed, add some documentation why.